blssignatures/aggregator.go

```diff
@@ -0,0 +1,80 @@
+package blssignatures
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/drand/kyber/sign"
+	"github.com/drand/kyber/sign/bdn"
+	"github.com/filecoin-project/go-f3/f3"
+)
+
+// SigAggregator holds the intermediate state of signature aggregation.
+// Must not be allocated directly. Always use AggSignatureScheme.NewAggregator to create a new instance.
+type SigAggregator struct {
+	blsScheme  *BLSAggScheme
+	sigs       []indexedSig
+	signerMask *sign.Mask
```

Comment:
Unclear here what sigs and signerMask really are, comment.

Reply:
Does sign.Mask use run-length encoding or how is the list of signers stored/marshaled?